abcorp/src/components/booking/BtnBack.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+export const BtnBack = ({navigation}) => {
+  return (
+    <Icon
+      name="chevron-back-outline"
+      size={40}
+      color="#5D60F0"
+      onPress={() => navigation.goBack()}
+    />
+  );
+};
```

Comment:
unnecessary component  split you can call this button on the necessary string like
`import React from 'react';
import BtnBack from 'react-native-vector-icons/Ionicons';

export const Main component = ( {navigation,rest}) => {
  return (
    <BtnBack
      name="chevron-back-outline"
      size={40}
      color="#5D60F0"
      onPress={() => navigation.goBack()}
    />
  );
};`